ironfish-rust-nodejs/index.js

```diff
@@ -275,6 +275,10 @@ module.exports.DECRYPTED_NOTE_LENGTH = DECRYPTED_NOTE_LENGTH
 module.exports.Note = Note
 module.exports.TransactionPosted = TransactionPosted
 module.exports.PROOF_LENGTH = PROOF_LENGTH
+module.exports.TRANSACTION_SIGNATURE_LENGTH = TRANSACTION_SIGNATURE_LENGTH
+module.exports.TRANSACTION_PUBLIC_KEY_LENGTH = TRANSACTION_PUBLIC_KEY_LENGTH
+module.exports.TRANSACTION_EXPIRATION_LENGTH = TRANSACTION_EXPIRATION_LENGTH
+module.exports.TRANSACTION_FEE_LENGTH = TRANSACTION_FEE_LENGTH
```

Comment:
This isn't a transaction specific fee length, its a currency encoded size which we always treat as a U64, so I think we can just make 1 constant if you really wanted one to encode the length of a currency amount that is applicable to all currency amounts.

Reply:
Agreed with Jason, this name can be more generic for any sort of amount.

Reply:
I want to point out Fee has type of I64,  but its the same length as U64. 
I agree with one constant of number length, it will be used for all the number types, eg notes size, spends size, tree size, authPath length. Since these x_sizes are fixed type.

But I don't want to mix transaction fee and transaction expiration with the above, due to
- have the flexibility of changing fee and expiration type in the future without touching other number type 
- improve code readability. If we need to add a new field in transaction, we can easily identify all the places to add.

Reply:
I'm not sure if @NullSoldier was referring to conflating expiration length. I think we can just have a currency amount length (or some equivalent)

Reply:
Added TYPE_NUMBER_SIZE constants to represent the size of type `number` in typescript. This will be the generic one for all the number type size. I keep all the constants for transaction attribute since transaction attribute has its own types defined in rs package.